activities/Moon.activity/css/activity.css

```diff
@@ -179,4 +179,4 @@ canvas {
 	height: 35px;
 	background-color: #808080;
 	border: 2px solid #808080;
-}
+}
```

Comment:
This file should not be updated.

Reply:
I didn't update it it just shows as if it was, initially I thought it should be, you can see it's the same as dev banch